cg/store/api/status.py

```diff
@@ -634,8 +634,8 @@ def analyses_to_delivery_report(self, pipeline: Pipeline = None) -> Query:
             .filter(VALID_DATA_IN_PRODUCTION < models.Analysis.started_at)
             .join(models.Family, models.Family.links, models.FamilySample.sample)
             .filter(
-                or_(
-                    models.Family.data_analysis.is_(None),
+                and_(
+                    models.Analysis.pipeline == str(pipeline),
                     models.Family.data_analysis == str(pipeline),
                 )
```

Comment:
```suggestion
                models.Analysis.pipeline == str(pipeline),
```
This should also give a valid result?